lib/mappers/wisconsin_mapper.py

```diff
@@ -8,42 +10,129 @@ def __init__(self, provider_data):
 
     def map_data_provider(self):
         """//edm:dataProvider -> .dataProvider (required property)"""
-        self.mapped_data.update(
-            {'dataProvider': getprop(self.provider_data, 'dataProvider')})
+        if exists(self.provider_data, 'dataProvider'):
```

Comment:
As a general measure for efficiency, it's nice to use `x = getprop(self.provider_data, 'someField', True)`, which initializes `x` to `None` if the property doesn't exist, and then say `if x:` so that you don't have to inspect the data twice. (Inspect it once and grab the value if it exists at the same time.)  If it were a dict you could use `x.get('someProp')` or `x.get('someProp', 'my default')`.